src/components/GardensDashboard.js

```diff
@@ -1,124 +1,24 @@
-import React, { useCallback } from 'react'
-import { GU, shortenAddress, textStyle, useTheme } from '@1hive/1hive-ui'
-import { useHistory } from 'react-router'
-
-import { useGardens } from '../providers/Gardens'
-
-import defaultGardenLogo from '../assets/defaultGardenLogo.svg'
-import defaultTokenLogo from '../assets/defaultTokenLogo.svg'
+import React from 'react'
+import styled from 'styled-components'
+import GardensList from './GardensList'
+import LandingBanner from './LandingBanner'
+import { useNodeHeight } from '@hooks/useNodeHeight'
 
 function GardensDashboard() {
-  // TODO :  add loading component
-  const { gardens } = useGardens()
+  const [height, ref] = useNodeHeight()
 
   return (
-    <div
-      css={`
-        padding: ${3 * GU}px;
-        display: grid;
-        grid-gap: ${2 * GU}px;
-        grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
-      `}
-    >
-      {gardens.map((garden, index) => (
-        <GardenCard key={garden.id} garden={garden} />
-      ))}
+    <div>
+      <LandingBanner ref={ref} />
+      <DynamicDiv marginTop={height}>
+        <GardensList />
+      </DynamicDiv>
     </div>
   )
 }
 
-function GardenCard({ garden }) {
-  const theme = useTheme()
-  const history = useHistory()
-  const handleSelectGarden = useCallback(() => {
-    history.push(`/garden/${garden.address}`)
-  }, [garden, history])
-
-  return (
-    <div
-      onClick={handleSelectGarden}
-      css={`
-        padding: ${5 * GU}px ${4 * GU}px ${3 * GU}px ${4 * GU}px;
-        background: ${theme.surface};
-        border: 1px solid ${theme.border};
-        border-radius: ${2.5 * GU}px;
-        box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.15);
-        cursor: pointer;
+const DynamicDiv = styled.div.attrs(props => ({
```

Comment:
This is a suggestion done by styled-components.
Setting the margin through `css={``}` directly resulted in styled components creating a class for each value